src/levanter/config.py

```diff
@@ -93,26 +93,51 @@ def _maybe_get_config_path_and_cmdline_args(args: List[str]):
     If URL, we need to download it and save it to a temp file. We then want to remove --config_path
     from the cmdline args so that draccus doesn't try to load it as a config path and return it separately here
     along with the modified cmdline args.
+    We also accept ... --configs <config1> <config2> ... and concatenate them into a single config.
```

Comment:
Why are we not using something like argparse rather than doing command-line parsing ourselves?

Reply:
we are using argparse at the end of the day, but this is a hacky layer to support reading configs from gcs (and i guess now supporting multiple configs)

it'd be better to do this in draccus.

Reply:
@dlwh Let me know if we are doing this in draccus or levanter

Reply:
draccus is probably better